plugins/org.jkiss.dbeaver.ui.editors.connection/src/org/jkiss/dbeaver/ui/dialogs/driver/DriverDependenciesTree.java

```diff
@@ -142,8 +143,12 @@ public boolean loadLibDependencies() throws DBException {
         filesTree.removeAll();
         int totalItems = 1;
         for (DBPDriverDependencies.DependencyNode node : dependencies.getLibraryMap()) {
+            if (node.library.getType().equals(DBPDriverLibrary.FileType.license) && editable) {
```

Comment:
Same as the previous note

```suggestion
            if (editable && node.library.getType().equals(DBPDriverLibrary.FileType.license)) {
```